MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/MovieDetailsRepository.swift

```diff
@@ -40,3 +41,10 @@ class MovieDetailsRepository: MovieDetailsRepositoryType {
         coreDataManager.createItem(movieDetails: movieDetails)
     }
 }
+
+// MARK: - MovieDetailsRespository Extension
```

Comment:
I think i left a comment in the other PR that this doesn't need to be in an extension :)

Reply:
yea sorry bro, I'll move all the functions in extension as I touch them again 🙏